src/futils.cpp

```diff
@@ -369,6 +373,14 @@ std::string getProcessPath() {
     auto path = fs::read_symlink(Internal::stringFormat("/proc/%d/path/a.out", getpid()));
 #elif defined(__unix__)
     auto path = fs::read_symlink("/proc/self/exe");
+#elif defined(__APPLE__)
```

Comment:
does this work on macOS as well? If so, might as well replace the previous condition.

Reply:
It probably does ?!
Replace or switch positions? Not sure which other unixes use this branch.

Reply:
I believe PROC_PIDPATHINFO_MAXSIZE is only for macOS.

Reply:
I'll move it up then 👍